libxi will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxi it will fail in the following way:

```
2024/11/12 23:12:47 INFO running step "test/pkgconf"
2024/11/12 23:12:47 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:47 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:47 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:47 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:47 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + basename /home/build/melange-out/libxi-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + dev_pkg=libxi-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + apk info -L libxi-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + grep -q ^Name: usr/lib/pkgconfig/xi.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + basename usr/lib/pkgconfig/xi.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + lib_name=xi uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + echo usr/lib/pkgconfig/xi.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + grep -q '^usr/lib/pkgconfig/xi.pc$\|^usr/share/pkgconfig/xi.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + pkgconf --exists xi uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN + pkgconf --print-errors --exists xi uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN Package inputproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN Perhaps you should add the directory containing `inputproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 WARN Package 'inputproto', required by 'xi', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:47 INFO pod 5d2f2e40d674108e0162adc6df6c9e531a72bbfc47ef5d98d6bac6500fe396ef terminated
2024/11/12 23:12:47 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:47 INFO   workspace dir: /home/user/tmp/melange-workspace-2407683926
2024/11/12 23:12:47 INFO   guest dir: /home/user/tmp/melange-guest-2650180212
2024/11/12 23:12:47 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxi] Error 1
```

--- a/libxi.yaml
+++ b/libxi.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxi
   version: 1.8.2
-  epoch: 0
+  epoch: 1
   description: X11 Input extension library
   copyright:
     - license: MIT AND X11
@@ -54,7 +54,11 @@ subpackages:
       runtime:
         - libxi
         - libxfixes-dev
+        - xorgproto
     description: libxi dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxi-doc
     pipeline:

